main_server.py

```
@@ -0,0 +1,445 @@
+            <p><em>Service Status: ✅ Online</em></p>
+        </div>
+    </body>
+    </html>
+    """
+    return HTMLResponse(content=html_content)
+
+# Priority Endpoint 1: AnalysePosts
+@app.post("/AnalysePosts", response_model=AnalysisResponse)
+async def analyse_posts(
+    text: Optional[str] = Form(None),
+    files: List[UploadFile] = File(default=[]),
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)